dev/intro-two.ftd

```diff
@@ -0,0 +1,128 @@
+
+-- ftd.type heading-hero-bold: $fpm.type.heading-hero.font
```

Comment:
@priyanka9634 do not change type of major elements from `fpm.type` rather define custom fonts on such cases!